natgeo/mortar
# Feature/SYTO-234

https://jira.nationalgeographic.com/browse/SYTO-234

This PR removes form and generic styles from Member Center and adds them to Mortar. Examples will need to be added to Mortar to capture some of the newer styles, but that seems out of scope for this ticket, at this point.

[ADDED] Styles from MC, which may or may not belong in Mortar
[MODIFIED] Moved font family styles to appropriate places in _typography.scss
[MODIFIED] Converted type-buttons,-default to mt_font-serif-stack,-sans,-italic
[REMOVED] Deleted non-relative font-size attributes
[MODIFIED] Moved colors from shame to colors, where appropriate
[REMOVED] Extraneous grey declaration, in favor of existing, more generic color
[MODIFIED] Moved Facebook styles from shame to _btn.scss
[MODIFIED] Swapping out color-specific values for generic ones, before eliminating them entirely
[MODIFIED] Trying hex for FB blue because hl was producing weird results.
[MODIFIED] Adjusting hex values to be accurate to the hex
[REMOVED] Took the shame.scss away
[ADDED] Made color.scss to allow access to mt_color-gamma
[ADDED] Actually importing the color settings. Logical.
[MODIFIED] Fixing naming of colors
[ADDED] Border modifier class to btn.scss
[ADDED] Created _form.scss and added .mt_input-form
[ADDED] Mixin and form to mortar.scss
[ADDED] Cursor pointer to button hover states.
[ADDED] Proper hover states for buttons

To Test:
- Pull branch
- Run `grunt serve`
- Make sure nothing is broken on Mortar
- :chart_with_upwards_trend:
